server.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+
+app.get("/", function (req, res) {
+  res.send("Hello World");
```

Comment:
And it begins!!!!!

Reply:
Yay onto new challenges and learnings ...woot woot